.pre-commit-config.yaml

```diff
@@ -82,13 +55,8 @@ repos:
   - repo: https://github.com/pre-commit/pygrep-hooks
     rev: v1.10.0
     hooks:
-      - id: python-check-blanket-noqa
-      - id: python-check-blanket-type-ignore
-      - id: python-no-log-warn
-        exclude: ^tests/test_sessions.py$
```

Comment:
Ruff's implementation is smarter than a simple grep, so it can tell session.warn isn't logging's warn! :)